bin/handlerFunctions.js

```diff
@@ -1406,15 +1481,19 @@ async function requestFetch(requestOptions, user, pass, filePath, succReturnCont
             return await response.json();
         }
       } else {
+        let jsonPayload = await response.json();
+        let message = jsonPayload?.data?.error?.message ? jsonPayload?.data?.error?.message : 'Response Error';
         throw new ApiError(
-          'Response Error',
+          message,
           response.status,
           response.headers.get('content-type')
         );
       }
     } else {
+      let jsonPayload = await response.json();
+      let message = jsonPayload?.data?.error?.message ? jsonPayload?.data?.error?.message : 'Unexpected content type in response';
```

Comment:
this can just be `jsonPayload?.data?.error?.message  || 'Unexpected content type in response'`

Reply:
Updated in https://github.com/brightsign/player-cli/pull/23/commits/91ba4af98ae55584cce43c3e741d7a8fb49e674c.